src/components/StyledInput.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, HTMLAttributes, useState } from "react";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors } from "styles/palette";
+
+interface Props {
+  value: string;
+  error?: string;
+  focused?: boolean;
+  onBlur?: () => void;
+  onFocus?: () => void;
+  label?: string;
+  placeholder?: string;
+  style?: HTMLAttributes<HTMLInputElement>;
+  type?: "submit" | "text" | "number";
+}
+
+const StyledInput: FC<Props> = ({
+  error,
+  focused = false,
+  label,
+  onBlur,
+  onFocus,
+  placeholder,
+  style,
+  type = "text",
+  value = "",
```

Comment:
No. Value is not an optional property so you shouldn't set default value for this.